Assets/Scripts/Board/Coord.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+
+public struct Coord
+{
+    public readonly int x;
+    public readonly int y;
+       
+    public Coord(int x, int y)
+    {
+        this.x = x;
```

Comment:
this sera inutile après le changement précédent

Reply:
je ne vois pas pourquoi